feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/rrm_config.c

```diff
@@ -200,14 +260,9 @@ int rrm_get_backup_channel(const char * freq_band)
 
 }
 
-bool rrm_noreload_config(struct schema_Wifi_RRM_Config *conf, const char *rname)
+bool rrm_noreload_config(struct schema_Wifi_RRM_Config *conf, const char *wlanif)
```

Comment:
Not really part of this PR. But I think we can change this function to void, since we aren't actually returning anything else other than true 

(it looks like that if statement that this was called in before didn't even have anything in it in the first place)

Reply:
Done